lang/shared/src/main/scala/com/wavesplatform/lang/v1/evaluator/ctx/impl/PureContext.scala

```diff
@@ -514,6 +514,21 @@ object PureContext {
       )
   }
 
+  lazy val makeString: BaseFunction[NoContext] =
+    NativeFunction("makeString", 30, MAKESTRING, listString, ("list", LIST(STRING)), ("separator", STRING)) {
+      case ARR(list) :: CONST_STRING(separator) :: Nil =>
+        val expectedStringSize =
```

Comment:
Could we check limits by array weight without calculation utf8 bytes?

Reply:
Fixed